ovh/helpers/helpers.go

```diff
@@ -256,6 +256,23 @@ func GetNilIntPointerFromData(data interface{}, id string) *int {
 	return nil
 }
 
+func GetNilFloat64PointerFromData(data interface{}, id string) (*float64, error) {
+	if resourceData, tok := data.(*schema.ResourceData); tok {
+		if val, ok := resourceData.GetOk(id); ok {
+			return GetNilFloat64Pointer(val)
+		}
+	} else if mapData, tok := data.(map[string]interface{}); tok {
+		if val, ok := mapData[id]; ok {
+			return GetNilFloat64Pointer(val)
+		}
+	} else {
```

Comment:
`else` not needed in that case, you can directly return the error

Reply:
not here, i must use an else condition